rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -3109,20 +3109,21 @@ rmw_destroy_wait_set(rmw_wait_set_t * wait_set)
   return RMW_RET_OK;
 }
 
+//==============================================================================
 static bool has_triggered_condition(
-  rmw_subscriptions_t * subscriptions,
-  rmw_guard_conditions_t * guard_conditions,
-  rmw_services_t * services,
-  rmw_clients_t * clients,
-  rmw_events_t * events)
+  const rmw_subscriptions_t * const subscriptions,
+  const rmw_guard_conditions_t * const guard_conditions,
+  const rmw_services_t * const services,
+  const rmw_clients_t * const clients,
+  const rmw_events_t * const events)
```

Comment:
With the latest changes here, I think this entire function could be const, i.e.:

```suggestion
  const rmw_events_t * const events) const
```

Reply:
Ah, never mind, this is a free function, so can't be const.